suspect/processing/channel_combination.py

```diff
@@ -3,14 +3,19 @@
 
 def svd_weighting(data):
     p, _, v = numpy.linalg.svd(data, full_matrices=False)
+    
     channel_weights = p[:, 0].conjugate()
+    # normalize so they sum to 1
+    channel_weights = channel_weights/numpy.sum(numpy.abs(channel_weights))
 
     # try some basic phase correction
     # in our truncation of the SVD to rank 1, we know that v[0] is our FID
     # use the first point of it to phase the signal
     phase_shift = numpy.angle(v[0, 0])
+    
+    channel_weights_phase_corrected = channel_weights * numpy.exp(-1j * phase_shift)        
 
-    return channel_weights * numpy.exp(-1j * phase_shift) / numpy.sum(numpy.abs(channel_weights))
+    return channel_weights_phase_corrected, channel_weights, phase_shift
```

Comment:
I think that this is unnecessary - the phase shift here is completely insignificant as there is going to be some proper phase correction happening later on anyway, this is just a very rough and ready attempt to put the signal in phase to make it easier to see what is going on when plotting e.g. to check for frequency drift. The point is that the channel weights are only defined up to an arbitrary phase, so we have to choose something and this makes the most sense - it is actually better than doing nothing in all cases.